File: app/src/main/java/au/id/blackwell/kurt/lantv/player/TvMediaPlayerListener.java
package au.id.blackwell.kurt.lantv.player;

interface TvMediaPlayerListener {
    void onPrepared();

    void onBufferingUpdate(int progress);

    boolean onError(int what, int extra);

    void onVideoSizeChanged(int width, int height, float aspectRatio);

    boolean onInfo(int what, int extra);

    void onCompletion();
}

File: app/src/main/java/au/id/blackwell/kurt/lantv/player/TvMediaPlayerListener.kt
package au.id.blackwell.kurt.lantv.player

interface TvMediaPlayerListener {
    fun onPrepared()

    fun onBufferingUpdate(progress: Int)

    fun onError(what: Int, extra: Int): Boolean

    fun onVideoSizeChanged(width: Int, height: Int, aspectRatio: Float)

    fun onPlaying()

    fun onCompletion()
}
